Pipfile

```diff
@@ -21,6 +21,7 @@ uvloop = "*"
 websockets = "*"
 httptools = "*"
 typing-extensions = "*"
+build = "*"
```

Comment:
I think if we put this into the Pipfile it should go under dev-packages

Reply:
You right. I don't think it's actually needed... right? I think I installed it when I was having trouble uploading to pypi.

Reply:
Yeah, fine to exclude entirely -- more of an "administrative" dependency (for whoever uploads to Pypi) than a dev dependency